lib/features/reps_core/reps_core.module

```diff
@@ -843,3 +843,18 @@ function reps_core_form_page_node_form_alter(&$form, &$form_state, $form_id) {
     $form['body']['#weight'] = -2;
   }
 }
+
+/**
+ * Implements hook_views_pre_render().
+ */
+function reps_core_views_pre_render(&$view) {
+  if ($view->tag == 'nodequeue') {
+    if (user_is_anonymous()) {
+      drupal_not_found();
```

Comment:
you'd better use https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_access_denied/7.x